python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -249,6 +262,12 @@ def __init__(self, butler=None, **kwargs):
         if self.config.doMeasurement:
             self.makeSubtask("measurement", schema=self.schema,
                              algMetadata=self.algMetadata)
+        if self.config.doForcedMeasurement:
+            self.totFluxKey = self.schema.addField(
+                "totFlux", "D", "Forced flux measured on the PVI")
```

Comment:
the name of these keys is not very descriptive, and does not seem to match the documentation exactly. I assume tot means total? How does this "total" relate to the forced flux measured? Does the forced flux give a total in some way that is clearer if you know about PVI (it may just be my ignorance). Consider changing the name to something more descriptive (or just like pviFlux etc), or changing the documentation string to clarify. I would say add a comment here, which may not be a bad idea, but that will not be documentation to someone who later just sees the schema and not what produced it.

Reply:
Just for reference, since I happened to notice this whizzing by — @morriscb is following the naming convention established in the DPDD (LSE-163, bottom of page 18). I don't like the name much — @natelust is right to flag it — but I agree with Chris that using it here is appropriate.